CONTRIBUTING

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+Hey, thanks for your interest in contributing to the NYU CS Wiki! Please review this document **before making a pull request**.
+
+## Guidelines 
+
+- All articles are written in a format known as [Markdown](https://commonmark.org/help/). Don't be worried! It's very simple to write in once you get the hang of it; for reference, you can look at any of the pages in the /docs folder. 
+- Looking in /docs, there are 3 main categories: Academics/Opportunities/Recreation. Let's take a look at /docs/Academics for reference. You can see an ``academics.md`` file which serves as an overview (visible when clicking on a category). From here, there is ``classes.md``, ``schedule.md``, etc which talk about their respective subjects. However, if there's a topic that requires multiple articles, you can create a folder (ex. /Academics/Tandon) and create an overview file and more pages from here. 
+- Let's take a look at one file: ``clubs.md``. At the top of the page are the title, tags, and sidebar position; you can add tags that you feel are relevant to the article (in this case, CAS, Tandon, and Clubs) - you can go to /docs/tags to view all current tags or you can create your own. The sidebar position isn't usually relevant but it can change the position of the article on the sidebar independent of alphabetical sorting.
```

Comment:
It might be good to provide a link to the file (or copy the piece of text here).